internal/install/recipe_installer.go

```diff
@@ -231,6 +236,25 @@ func (i *RecipeInstaller) Install() error {
 		log.Debugf("Done installing integrations.")
 	}
 
+	isAppTarget := func(recipe types.Recipe) bool {
```

Comment:
Maybe worth a convo about this pattern in general, but would this entire func be a bit more readable with the helper in its own scope?